include/stl2/detail/range/concepts.hpp

```diff
@@ -265,7 +265,7 @@ STL2_OPEN_NAMESPACE {
 		template<class Rng>
 		concept bool ViewableRange =
 			Range<Rng> &&
-			(std::is_lvalue_reference_v<Rng> || View<__f<Rng>>);
+			(requires{typename __begin_t<Rng>;} || View<__f<Rng>>);
```

Comment:
`__begin_t` should be `__end_t` here, or require `Sentinel<__end_t<Rng>, __begin_t<Rng>>` directly.